### ISSUE TITLE
[Bug]: [PartitionKey] should retrun error if set multiple partition key fields

### ISSUE THREAD

**Issue by yanliang567** [milvus-io/milvus#24154]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.2.0-20230515-8b8f9f1e
- Deployment mode(standalone or cluster):
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus-2.2.9.dev8
```

### Current Behavior

create collection schema successfully with parition_key_field=[field_1, field_2]

### Expected Behavior

return an error 

### Steps To Reproduce

```markdown
1. create a collection schema with multiple parition key fileds. (parition_key_field=[f1,f2])
```

**Comment by xige-16** [comment]
fix pr https://github.com/milvus-io/pymilvus/pull/1441

**Comment by yanliang567** [comment]
verified fix on pymilvus-2.2.9.dev11
